python_examples/gibraltar_example/simulation_outputs/gibraltar_weeky_cases.csv

```diff
@@ -1,14 +1,14 @@
 dates,0-10,10-20,20-30,30-40,40-50,50-60,60-70,70-80,80-90
-03-18,8.73478641368077,15.087127972428986,12.643181019311266,15.270662637622117,23.827039560958518,19.821590509655632,19.7655371645609,14.87284215873912,15.377643556537912
-03-25,23.03139880200083,55.07337183015331,49.440765036993696,62.73672967240551,99.50453427035772,99.0035623427829,92.78382787540676,71.96703776950841,66.3747845228265
-04-01,71.99266453838958,193.9619703377142,162.11955525220938,201.7380814302143,285.52255695912515,318.7608473761094,289.2855003287369,236.9164884354549,248.1608722769091
-04-08,123.41325275244235,328.52320540786246,309.49342598813223,333.7566943276437,537.7955382125192,544.3861734685886,525.3048171316626,492.8133844911901,480.96168263361386
-04-15,142.72426903594632,374.7858848309659,385.5001702470392,404.2893881384403,663.1584510152945,674.3980391249866,660.9832808859163,587.2083456381492,634.4226178334704
-04-22,139.24444585888727,355.95593038192027,348.344795221689,371.0741325840669,612.3694976668265,664.5081991767101,591.619336026812,520.3550323506345,607.151953704786
-04-29,116.89690559853368,288.71790520369825,252.8911888233648,269.91266135093053,473.38001703264405,549.7285262384479,465.36782262898856,401.11123623273846,534.1115415206164
-05-06,82.01764265972515,197.82197123452696,185.97507832946502,186.12273667131265,324.40655997412847,370.1388138150599,350.94178566184536,294.99141912150805,418.2204147146846
-05-13,56.31328341868284,125.37543326977625,116.75852587761706,133.14097023552887,226.15925756663358,236.4606722473898,264.71563051380394,216.03080193326073,244.41609717374354
-05-20,39.628777124127346,76.75755424825469,71.88918941205193,78.63967522673312,136.84565216017546,157.54903997066373,166.7540440818939,163.54838754576036,159.02243491234205
-05-27,24.913843656933384,51.60892417764053,60.721354147859216,52.02692138006031,86.66049950436984,107.92155855116117,104.79790451439541,127.8280069159424,82.48559272507494
-06-03,18.25080969113531,33.91449140984163,41.90456493659823,34.07240020079094,61.42506533778083,82.77325255580944,78.18035066158193,83.95068832930427,54.2460071004867
-06-10,9.32703956095852,15.280912987319644,24.14350489576539,21.959646354118135,56.031722280838345,60.349159443815886,56.88373867087848,56.21525495794846,40.873164643535134
+03-18,0.7194108891344781,3.6531074925546707,9.495198701605359,19.959646354118135,31.65407912191704,30.65407912191704,20.41929260883212,7.995198701605359,18.5669510500839
```

Comment:
These results looks quite significantly different, which concerns me if all your changes are just speeding up the code without changing the output. Do you know why this might be?

Reply:
These weren't changed by my code, they hadn't been updated in a while

Reply:
So this would be the same if you reran the workflows on main? I was wondering if you have done this

Reply:
I don't know but they looked like the plots Lara had when we were troubleshooting the problems with cloning everyone in the Gibraltar simulation

Reply:
Correction: Code had a problem with not inverting the distance weights should now be resolved and better

Reply:
Okay, if that is something Lara has changed then we can wait until her PR is merged, but there shouldn't be any changes in this PR relative to that one by the sounds of it